packages/components/pie-modal/src/index.ts

```diff
@@ -154,33 +154,60 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         }
     }
 
-    render () {
-        const {
-            heading,
-            headingLevel = 'h2',
-            size,
-            isFullWidthBelowMid,
```

Comment:
I quite like this destructuring to simplify the variables – was there a reason for removing it?

Reply:
After extracting the header and footer creation into their own functions, there were only two variables left that were being used.

By removing the destructuring, it lets us use the arrow-body syntax which I think helps dissuade people from adding extra logic into the render function, instead moving it out into a separate function.

What do you think?

Reply:
I think it's probably worth catching up with @jamieomaguire as well when he's in tomorrow – he's done some restructuring of the render function in the `hasBackButton` PR as well which will definitely conflict with this, and I think we probably need to be careful not to abstract out everything into separate render functions (as I think it can have the opposite effect then whereby it makes things harder to find).

Think we likely will need to discuss and work out the best way to merge both of your PRs into main in the easiest way.